index.html

```diff
@@ -1491,6 +1469,29 @@ <h5 id="General Traits">General Traits</h5>
                <td>Global Level 3 or Nature Level 1
                </td>
             </tr>
+            <tr>
+               <td>Purity of Body
+               </td>
+               <td>
+                  When you take a short rest you can regain 5 resistance points of your choice instead of healing.
+               </td>
+               <td>
+               </td>
+               <td>Global Level 3
+               </td>
+            </tr>
+            <tr>
+               <td>Purity of Mind
+               </td>
+               <td>
+                  You can spend 5 resistance points of your choice to gain another 3AP this turn.
```

Comment:
I am 0% ok with this. No way José. I do not want to give literally everyone in the party a way to take almost 2 turns at the start of a fight to gain a big advantage.